core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -244,9 +244,12 @@ public enum Property {
   GENERAL_OPENTELEMETRY_ENABLED("general.opentelemetry.enabled", "false", PropertyType.BOOLEAN,
       "Enables tracing functionality using OpenTelemetry (assuming OpenTelemetry is configured).",
       "2.1.0"),
+  @Deprecated
```

Comment:
This should also have a `@ReplacedBy` annotation

Reply:
Fixed in commit b086f0234deac1b629713aa488a5fad2944de665